node/libs/roles/src/attester/tests.rs

```diff
@@ -0,0 +1,234 @@
+use crate::{attester, validator::testonly::Setup};
+
+use super::*;
+use assert_matches::assert_matches;
+use rand::Rng;
+use zksync_concurrency::{ctx, time};
+use zksync_consensus_crypto::{ByteFmt, Text, TextFmt};
+use zksync_protobuf::testonly::test_encode_random;
+
+#[test]
+fn test_byte_encoding() {
+    let ctx = ctx::test_root(&ctx::RealClock);
+    let rng = &mut ctx.rng();
+
+    let sk: SecretKey = rng.gen();
+    assert_eq!(
```

Comment:
SecretKey for all roles now implement PartialEq, so you can just directly compare sk with decode(encode(sk)). Please also update tests for other roles.